src/Plugin/migrate/source/OntologyWordDetails.php

```diff
@@ -69,6 +62,27 @@ public function fields() : array {
     return [];
   }
 
+  /**
+   * Gets the ontology ids that are included to the migration.
+   *
+   * Set ontology ID limit using 'helfi_tpr.limit_ontology_words:ids' config.
+   * If the config is not set, $defaultOntologyIdsLimit is used.
+   *
+   * If given an empty array, migrate does not limit by ontology ids.
+   *
+   * @return array
+   *   List of ontology ids.
+   */
+  public function getOntologyIdsLimit(): array {
+    if ($config = \Drupal::configFactory()->get('helfi_tpr.limit_ontology_words')) {
```

Comment:
Use dependency injection for this, for example:

````php
  /**
   * {@inheritdoc}
   */
  public static function create(
    ContainerInterface $container,
    array $configuration,
    $plugin_id,
    $plugin_definition,
    MigrationInterface $migration = NULL
  ) {
    $instance = new static($configuration, $plugin_id, $plugin_definition, $migration);
    $instance->configFactory = $container->get('config.factory');
    return $instance;
  }
````

or you can just populate `$instance->defaultOntologyIdsLimit` there as well.